mantidproject/mantid
# Highlight failed measurement transfers in Reflectometry GUI (Polref)

There were some changes added to the GUI in #14051 that allowed the user to search the ICAT service by `Measurement` as well as `Description`.

Some of the runs that were retrieving by the `Measurement` search could not be successfully transferred into the processing table of the GUI.

These runs are now highlighted in Orange and a tooltip is set describing why they could not be transferred into the processing table.
- [ ] **Documentation**: needs to explain the new transfer methods and what the highlighted rows mean.

**To Test**:
 You will need access to the ICAT service to perform this test.
- Go to Interfaces > Reflectometry > ISIS Reflectometry (Polref)
- In `Search Runs` change the `Instrument` to POLREF
- Enter the investigation ID: `1520249`
- Change transfer method from `Description` to `Measurement`
- Run Search
- Select any run before 14490, as they will fail
- Select any run after 14490, as they will transfer succesfully
- click Transfer
- Once the processing table is populated, click back on the Search Table
- See that any failed transfers will be orange and the tooltip (hover over cell) should explain why they failed

**Release notes**: to do.

Fixes #14370
